ui/components/app/confirm-page-container/confirm-page-container-header/confirm-page-container-header.component.js

```diff
@@ -64,7 +72,19 @@ export default function ConfirmPageContainerHeader({
             </span>
           </div>
         )}
-        {isFullScreen ? null : <NetworkDisplay />}
+        {isFullScreen ? null : (
+          <PickerNetwork
+            label={
+              networkType === NETWORK_TYPES.RPC
+                ? nickname ?? t('privateNetwork')
+                : t(getNetworkLabelKey(networkType))
+            }
+            backgroundColor={BackgroundColor.transparent}
```

Comment:
Let's update the border using the style utility props. It will save having to update the `PickerNetwork` component directly.
```suggestion
            backgroundColor={BackgroundColor.transparent}
            borderColor={BorderColor.borderMuted}
```